src/routes/Manual.svelte

```diff
@@ -76,27 +76,28 @@ function handleClickRfPower() {
 
 // MARK: Speciy Timer Config
 
-$: lfPeriod = 500;
+$: lfPeriod = 2000;
 $: lfDutyCycle = 50;
-$: hfPeriod = 250;
-$: hfDutyCycle = 100;
-$: single_pulse_duration = 500;
+$: hfPeriod = 200;
+$: hfDutyCycle = 50;
+$: single_pulse_duration = 1000;
+$: single_pulse_pause = 250;
 $: singlePulse = false;
 $: {
     if(singlePulse) {
-        $OP_Mode = 0x05;
+        $command = 0x05;
     } else {
-        $OP_Mode = 0x02;
+        $command = 0x02;
     }
    }
 
-$: $single_pulse_block = [single_pulse_duration & 0x000000ff,(single_pulse_duration & 0x0000ff00) >> 8,0,0];
+$: $single_pulse_block = [(single_pulse_duration & 0x00000ff0) >> 4,(single_pulse_duration & 0x0000000f) << 4 | (single_pulse_pause & 0x00000f00)>>8,single_pulse_pause & 0x000000ff];
 //set freq to 1kHz (start with 200Hz)
 $: hfOn = hfPeriod * (hfDutyCycle/100);
-$: $hf_block = [hfOn & 0x000000ff, (hfOn & 0x0000ff00) >> 8, hfPeriod & 0x000000ff,(hfPeriod & 0x0000ff00) >> 8];
+$: $hf_block = [(hfOn & 0x00000ff0) >> 4, (hfOn & 0x0000000f) << 4 | (hfPeriod & 0x00000f00)>>8, hfPeriod & 0x000000ff];
 
 $: lfOn = lfPeriod * (lfDutyCycle/100);
-$: $lf_block = [lfOn & 0x000000ff, (lfOn & 0x0000ff00) >> 8, lfPeriod & 0x000000ff, (lfPeriod & 0x0000ff00) >> 8];
+$: $lf_block = [(lfOn & 0x00000ff0) >> 4, (lfOn & 0x0000000f) << 4 | (lfPeriod & 0x00000f00)>>8, lfPeriod & 0x000000ff];
```

Comment:
It would be nice to comment on what the protocol is expecting. What is the intent of this bit math? For example, Grab the top bits of this and use it as the top byte of that.